projects/step-core/src/lib/directives/test-dataid.directive.ts

```diff
@@ -0,0 +1,14 @@
+import { Directive, ElementRef, inject, Input } from '@angular/core';
+
+@Directive({
+  selector: '[testDataId]',
```

Comment:
@neogucky @lukasz-lepa 
I think it make sense, to follow the naming convention and apply the selector with `step` prefix.
And make it like `[stepTestId]`